docs/source/admin/traffic_ops.rst

```diff
@@ -443,6 +447,8 @@ This file deals with the configuration parameters of running Traffic Ops itself.
 		:disabled_routes: A list of API route IDs to disable. Requests matching these routes will receive a 503 response. To find the route ID for a given path you would like to disable, run ``./traffic_ops_golang`` using the :option:`--api-routes` option to view all the route information, including route IDs and paths.
 		:ignore_unknown_routes: If ``false`` (default) return an error and prevent startup if unknown route IDs are found. Otherwise, log a warning and continue startup.
 
+	:tls_config: An optional stanza for TLS configuration. The values of which can be found here: https://golang.org/pkg/crypto/tls/#Config
```

Comment:
To link to a GoDoc, use our custom `:godoc:` role. In this case, it'd be:
```rst
The values of which conform to the :godoc:`crypto/tls.Config` structure.
```
That creates an automatic link.

Reply:
:+1: